nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,115 @@
+#ICA_AROMA pulled from: https://github.com/rhr-pruim/ICA-AROMA
+#This assumes ICA_AROMA.py is already installed and callable via $PATH
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    Directory,
+    traits,
+    OutputMultiPath
+)
+import os
+
+class ICA_AROMAInputSpec(CommandLineInputSpec):
+    featDir = Directory(exists=True,
+                        desc='If a feat directory exists and temporal filtering '
+                        'has not been run yet, ICA_AROMA can use the files in '
+                        'this directory.',mandatory=False,xor=['infile','mask','affmat','warp','mc'])
+    infile = File(exists=True, 
+                  desc='volume to be denoised', 
+                  argstr='-i %s',mandatory=False,xor=['featDir'])
+    outdir = Directory(desc='path to output directory', 
+                  argstr='-o %s',mandatory=True)
+    mask = File(exists=True, 
+                desc='path/name volume mask', 
+                argstr='-m %s',mandatory=False,xor=['featDir'])
+    dim = traits.Int(desc='Dimensionality reduction when running '
+                     'MELODIC (defualt is automatic estimation)', 
+                     argstr='-dim %d')
+    TR = traits.Float(desc='TR in seconds. If this is not specified '
+                      'the TR will be extracted from the header of the fMRI nifti file.',
+                      argstr='%.2f')
+    melodic_dir = Directory(exists=True,
+                            desc='path to MELODIC directory if MELODIC has already been ran',
+                            argstr='-meldir %s')
+    affmat = File(exists=True,
+                  desc='path/name of the mat-file describing the '
+                  'affine registration (e.g. FSL FLIRT) of the '
+                  'functional data to structural space (.mat file)',
+                  argstr='-affmat %s',mandatory=False,xor=['featDir'])
+    warp = File(exists=True,
+                desc='File name of the warp-file describing '
+                'the non-linear registration (e.g. FSL FNIRT) '
+                'of the structural data to MNI152 space (.nii.gz)',
+                argstr='-warp %s',mandatory=False,xor=['featDir'])
+    mc = File(exists=True,
+              desc='motion parameters file',
+              argstr='-mc %s',mandatory=False,xor=['featDir'])
+    denoise_type = traits.Str(argstr='-den %s',
```

Comment:
This should be an `Enum`, since we know all of the possible options.

```Python
denoise_type = traits.Enum('nonaggr', 'aggr', 'both', 'no', usedefault=True, desc='...')
```

I set `nonaggr` to the default here, since that seems consistent with the [source](https://github.com/rhr-pruim/ICA-AROMA/blob/master/ICA_AROMA.py#L42).